### PR TITLE
Update peer dependency of `react-relay` to accept version 0.7.2

### PR DESCRIPTION
**Pull request by nodkz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "react-static-container": "^1.0.0-alpha.1"
   },
   "peerDependencies": {
-    "react-relay": "0.7.1"
+    "react-relay": "^0.7.1"
```
Comment (denvned) [inline package.json:38]: _isomorphic-relay_ depends heavily on Relay internal API, that's why it uses exact version. But it's probably safe to write `"0.7.1 || 0.7.2"`. Have you tested it with 0.7.2?

Comment (nodkz) [inline package.json:38]: Yep. In my app works perfectly.

Comment (denvned) [inline package.json:38]: Thanks for confirming. Added 0.7.2 and 0.7.3.

**Comment by nodkz** [comment]
Great thanks for 0.5.4!

Got clean shrinkwrapped packages for my project.

**Comment by thelordoftheboards** [comment]
Thank you!

**Comment by fenos** [comment]
Thank you!

**Pull request closed without merging**